src/guide/getting_started.md

```diff
@@ -62,4 +67,72 @@ if let Some(node_state) = sdk.node_info()? {
     let balance_onchain = node_state.onchain_balance_msat;
 }
 ```
+</section>
+<div slot="title">Swift</div>
+<section>
+
+The first step is to register a new node
+## Registering a new node
+```swift
+do {
+ let seed = try mnemonicToSeed(phrase: "<mnemonics words>");
```

Comment:
NIT: Adding a semicolon at the end of lines in Swift is optional and generally not preferred. Think we can remove them. 
https://github.com/kodecocodes/swift-style-guide#semicolons